x-pack/plugins/cases/README.md

```diff
@@ -140,6 +171,57 @@ Arguments:
 UI component:
 ![Recent Cases Component][recent-cases-img]
 
+### hooks
+
+
+#### getUseCasesAddToNewCaseFlyout
+
+Returns an object containing two methods: `open` and `close` to either open or close the add to new case flyout.
+You can use this hook to prompt the use to create a new case with some attachments directly attached to it. e.g.: alerts or text comments.
+
+
+Arguments:
+
+| Property          | Description                                                                                                        |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------ |
+| userCanCrud       | `boolean;` user permissions to crud                                                                                |
+| onClose           | `() => void;` callback when create case is canceled                                                                |
+| onSuccess         | `(theCase: Case) => Promise<void>;` callback passing newly created case after pushCaseToExternalService is called  |
+| afterCaseCreated? | `(theCase: Case) => Promise<void>;` callback passing newly created case before pushCaseToExternalService is called |
+| attachments?      | `CaseAttachments`; array of `SupportedCaseAttachment` (see types) that will be attached to the newly created case  |
+
+returns: an object with `open` and `close` methods to open or close the flyout.
+
+`open()` and `close()` don't take any arguments. They will open or close the flyout at will.
+
+#### `getUseCasesAddToExistingCaseModal`
+
+Returns an object containing two methods: `open` and `close` to either open or close the  case selector modal.
+
+You can use this hook to prompt the use to select a case and get the selected case. you can also pass attachments directly  and have them attached to the selected case after selection. e.g.: alerts or text comments.
```

Comment:
nit: `the use` -> `the user`

Reply:
Also capitalize `you` at start of second sentence.